Odata-docs/webapi/pagesize-top-maxtop.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "PageSize, Top and MaxTop"
+description: "This tutorial describes how to configure PageSize, Top and MaxTop."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 3/1/2022
+---
+# PageSize, Top and MaxTop
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v6.md)]
+
+In OData WebApi we support both client driven and client side paginations.
```

Comment:
I think what you intended to say here is that we support both server-driven and client-driven paginations, no?

Reply:
Exactly.
Fixed